tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/BasicCasesTest.java

```diff
@@ -41,98 +41,112 @@ public void setup() throws BallerinaTestException {
 
     @Test
     public void testAssertions() throws BallerinaTestException {
-        String errorOutput = balClient.runMainAndReadStdOut("test", new String[]{"assertions"},
+        String errorOutput = balClient.runMainAndReadStdOut("test",
```

Comment:
can we use `@BeforeMethod` annotation to avoid putting this in each method ?

Reply:
Couldn't use a BeforeMethod since the rest of the args to the command are only available within the test and hence we cannot merge the args. Wrote a protected method and used it to prefix the coverage args.